microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/jaxrs/java/JaxRsCodeLensParticipant.java

```diff
@@ -137,7 +172,59 @@ private static void collectURLCodeLenses(IJavaElement[] elements, String rootPat
 		}
 	}
 
-	private static boolean isServerAvailable(String host, int port, int timeout) {
+	private static String validateInterfaceType(IType interfaceType,

```

Comment:
Rename to `findApplicationPathValue` and Javadoc in this method by given a sample with ApplicationPath and by saying that we scan Java sources project to collect this ApplicationPath

Reply:
Remove too interfaceType and context to have this signature:

`private static String findApplicationPathValue (IJavaProject javaProject)`

which is very understandable : search application path valuue inside the given Java project.